crates/ibc/src/core/ics02_client/handler.rs

```diff
@@ -17,19 +12,3 @@ pub enum ClientResult {
     Upgrade(upgrade_client::UpgradeClientResult),
     Misbehaviour(misbehaviour::MisbehaviourResult),
 }
```

Comment:
The `ClientResult` enum appears to be redundant. Does it have a future use? 
Otherwise, each variant in the respective module should be deleted as well

Reply:
[ff3c0f7](https://github.com/cosmos/ibc-rs/pull/456/commits/ff3c0f736586393ea740861f9d411fec619af838)